spec/classes/init_spec.rb

```diff
@@ -1661,6 +1663,25 @@
     end
   end
 
+  context 'with sshd_kerberos_authentication set to invalid value on valid osfamily' do
```

Comment:
Apologies for the spec testing code here.. it's a mess. Would you mind adding a different describe section something like this

``` ruby
describe 'with KerberosAuthentication' do
  ['yes','no'].each do |value|
    context "set to #{value}" do
      # set param to value
      it { should contain_file('sshd_config').with_content(/^KerberosAuthentication #{value}$/) }
    end
  # put this new code where you set it to invalid here
  end
end
```